src/onegov/ballot/models/election/election.py

```diff
@@ -44,6 +44,8 @@ class Election(Base, ContentMixin, TimestampMixin,
         'polymorphic_identity': 'majorz'
     }
 
+    i18n_used_locales = ('de_CH', 'fr_CH', 'it_CH', 'rm_CH')
```

Comment:
Wouldn't it make sense to store this at a single location? Maybe in something like `onegov/ballot/constants.py`. Bonus points for maybe fusing it with what's already in the `get_i18n_used_locales` function in `onegov/election_day/app.py`.

Especially because this seems to be duplicated many times over in other parts of this pull request.

Reply:
I tried to get this from the app since I saw it there, but accessing the app there inside the model was not possible. I'll move this to one file.